src/pybind/mgr/dashboard/controllers/user.py

```diff
@@ -113,6 +123,35 @@ def set(self, username, password=None, name=None, email=None, roles=None,
         mgr.ACCESS_CTRL_DB.save()
         return User._user_to_dict(user)
 
+    @Endpoint('POST')
+    @CreatePermission
```

Comment:
We can't limit the permissions of who calls this, otherwise a read only user won't be able to change password.
```suggestion
```